packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -864,13 +873,15 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                     // If the cache has the hash of the blob and handle of last summary is also present, then use that
                     // to generate complete path for the given blob.
                     if (!completePath || !this.lastSummaryHandle) {
+                        blobs++;
```

Comment:
Should this be "newBlobs" or something to explain that it does not include reusedBlobs?